cli/mender-artifact/modify.go

```diff
@@ -43,13 +41,8 @@ func modifyArtifact(c *cli.Context) error {
 		return cli.NewExitError("File ["+c.Args().First()+"] does not exist.", 1)
 	}
 
-	pubKey, err := processModifyKey(c.String("key"))
-	if err != nil {
-		return cli.NewExitError("Error processing private key: "+err.Error(), 1)
-	}
-
 	modifyCandidates, isArtifact, err :=
-		getCandidatesForModify(c.Args().First(), pubKey)
+		getCandidatesForModify(c.Args().First(), nil)
```

Comment:
It looks like all calls to this function pass either `nil` or empty buffer now. Remove the argument?

Reply:
:+1:

Reply:
Well, it is also used by `virtualPartitionFile.Open`, which is used for cp, cat, install and rm. So it falls into the same category as you other comment in `repackArtifact`: to verify the actual usage of the key in these operations and decide.

Reply:
Fixed. See last commit.